test/addon_runners/chrome/main.js

```diff
@@ -84,7 +90,13 @@ exports.testChromeInPanel = function(assert, done) {
     });
     panel.port.emit('echo');
   });
-  panel.show();
+
+  // NOTE: hack needed to prevents random test timeout (probably due to the
+  // setTimeout used in the 'panel.show' method:
+  // https://github.com/mozilla/addon-sdk/blob/a19bd8c/lib/sdk/panel/utils.js#L78)
+  intervalId = setInterval( _ => {
```

Comment:
looks like a good enough approach! :) My only worry is that if `show()` is never called then this interval would live on even if the test times out. Or will it die along with the timeout? If there's no way to address that then no worries, but here are some ideas:

* is there a `afterEach()` or `tearDown()` hook that can be used here to clear the interval?
* should the interval give up after several tries?